src/components/Typography/BodyX/BodyX.stories.tsx

```diff
@@ -73,14 +73,14 @@ export const All = {
     children: { table: { disable: true } },
   },
   render: (args: object) => (
-    <>
+    <div style={{ display: 'flex', flexDirection: 'column', gap: 16 }}>
```

Comment:
I see this is a repeating pattern. 
Can we create some mocks (e.g. one long text and one standard length) to be reused in tests and stories?